beanstalkworker/rawJob.go

```diff
@@ -88,6 +88,11 @@ func (job *RawJob) GetTube() string {
 	return job.tube
 }
 
+// GetConn returns the beanstalk connection
```

Comment:
thanks @NevilleC can you slightly change this comment, other than that its good to go.

"// GetConn returns the beakstalk connection used to receive the job."

Added full stop to end and clarified that this is the connection for the current job (As I am planning to modify bsworker to use multiple connections and concurrent workers).